mapstore2_adapter/plugins/serializers.py

```diff
@@ -133,13 +122,13 @@ def decode_base64(data):
         _map_thumbnail_format = 'png'
         if attributes:
```

Comment:
see my previous comment about attributes

Reply:
@mattiagiupponi let's do this additional step, so that we don't need to do anything more on the FE (in theory!),

The serializer will use the properties from the body payload and stop pulling title, abstract and thumbnail from the attributes.
The client will use the `thumbnail_url` property even though a base64 encoded image is passed in.

@mattiagiupponi I would consider moving t[his conversion](https://github.com/GeoNode/geonode-mapstore-client/blob/90c013a255d98dcec0d8cd45ad5f5f733ac1c741/mapstore2_adapter/plugins/serializers.py#L279-L287) to the core Map API v2 serializer, rather then keppeing it in the serializer. It might be useful for other use cases too, what do you think?